examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -7073,7 +7073,7 @@ endpoint 1 {
     ram      attribute coupleColorTempToLevelMinMireds;
     persist  attribute startUpColorTemperatureMireds;
     ram      attribute featureMap default = 0x1F;
-    ram      attribute clusterRevision default = 5;
+    ram      attribute clusterRevision default = 6;
```

Comment:
This should not be done to land before 1.2 tag, since otherwise someone going to SVE could claim Rev 6 support. Please do not change until we are opened up for 1.3

Reply:
@tcarmelveilleux 
The GitHub issue is targeted for 1.2 milestone and was reported in may.

https://github.com/project-chip/connectedhomeip/issues/26598

From what I understood test event instruction specifies the cluster revision should be 6 and reports the test a failure with 5.

If this can be clarified then I don't have an issue with delaying this.

Reply:
I confirmed with @lpbeliveau-silabs that the clarifications made to the cluster spec with revision 6, do not impact the color control cluster behaviour. The actions are done by the scene cluster implementation. Therefore we can say that the current colour control clusters respect its revision 6 specifications.